README.md

```diff
@@ -29,16 +29,18 @@ pyenv global 3.8.1
 python --version # Python 3.8.1
 ```
 
-Installing packages and managing virtual environments can be easily done with [pipenv](https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv):
+Project dependencies can be installed with `poetry`, `pipenv` or `pip`.
 
 ```bash
 # Upgrade pip and setuptools to ensure we can build libraries against 3.8
 python -m pip install --upgrade pip setuptools
 
 # Install pipenv and project dependencies
-python -m pip install pipenv && pipenv install
+python -m pip install poetry && poetry install
```

Comment:
If we use poetry instead of pipenv, shouldn't the areas of the README that use pipenv to run/install things be changed to use poetry as well?

Reply:
Whoops. The rest of the readme's been updated to refer to poetry!